### ISSUE TITLE
Add 'Copyright' file

### ISSUE THREAD

**Issue by deekej** [fedora-sysv/initscripts#78]
We should add `Copyright` which states that the files in this git repository that are under Red Hat's Copyright are actually distributed under GPLv2 [`COPYING`]. We need a clarification, what actually applies for the initscripts files.

Right now, we don't have such a thing.

**Comment by lnykryn** [comment]
https://github.com/fedora-sysv/initscripts/blob/master/COPYING ?

**Comment by deekej** [comment]
No, that's not what I meant. I have an example from ghostscript source code, but I will add the link for it later, once I will be dealing with this.
